### ISSUE TITLE
Loss value

### ISSUE THREAD

**Issue by reutlanyado** [maiminh1996/YOLOv3-tensorflow#18]
Hi!
Thanks for your great project.
I get Nan/inf when I'm running the training (just for 5 images from all my training data) , do you know way?
I get loss value of 125, for 21 box in the image, is it good?
I would like to get your training data in order to check the algorithm .

**Comment by YunYang1994** [comment]
why not try a smaller learning rate ?
maybe this repo can help you. https://github.com/YunYang1994/tensorflow-yolov3
